locations/spiders/petrol_bg.py

```diff
@@ -12,4 +13,18 @@ def parse_item(self, item, location):
         if m := re.match(r"^(\d+) (.+)$", item["name"]):
             item["ref"] = m.group(1)
             item["name"] = m.group(2)
+
+        categories = (location["categories"] or "").split(",")
+        apply_yes_no(Fuel.DIESEL, item, ("19" in categories or "20" in categories))
+        apply_yes_no(Fuel.OCTANE_100, item, "21" in categories)
+        apply_yes_no(Fuel.OCTANE_95, item, "22" in categories)
+        apply_yes_no(Fuel.LPG, item, "23" in categories)
+        apply_yes_no(Fuel.CNG, item, "24" in categories)
+        apply_yes_no(Extras.CAR_WASH, item, "25" in categories)
+        apply_yes_no(Extras.ATM, item, "26" in categories)
+        apply_yes_no("restaurant", item, "27" in categories)
+        apply_yes_no("self_service", item, "28" in categories)
+        apply_yes_no(Fuel.ADBLUE, item, "29" in categories)
+        apply_yes_no("amenity:chargingstation", item, "31" in categories)
```

Comment:
@Cj-Malone has some insight on desired tagging of charging stations/fuel stations at #5833 which may be helpful here.

Reply:
The spider now generates a new POI for the charging station.